docker-compose.yml

```diff
@@ -12,7 +12,7 @@ x-common-variables: &wikibase_variables
   DB_USER: ${DB_USER:-sqluser}
   DB_PASS: ${DB_PASS}
   DB_NAME: ${DB_NAME:-my_wiki}
-  WIKIBASE_HOST: ${WIKIBASE_HOST:-portal.mardi4nfdi.de}
+  WIKIBASE_HOST: ${WIKIBASE_HOST}
```

Comment:
removed it since this is not overwritten by values in env file.

Reply:
The way things are deployed, all defaults should be in docker-compose.yml. So if you remove this, WIKIBASE_HOST will not be set on the production instance.